exercises/stack_01.js

```diff
@@ -0,0 +1,82 @@
+window.Collections = {};
+
+(function(module) {
+    function Stack(arr) {
+        /* constructor */
+        this.arr = arr;
+    };
+
+    /* creating a forEach method on Stack */
+    Stack.prototype.forEach = function(callback) {
+        for (var i = 0; i < this.arr.length; i++) {
+            callback(this.arr[i], i, this.arr);
+        }
+    };
+
+    Stack.prototype.push = function(elem){
+        // this.arr.push(elem);
+        this.arr[this.arr.length] = elem;
+    }
+
+    Stack.prototype.peek = function() {
+        return this.arr[this.arr.length-1];
```

Comment:
Good. Although, for style, I'd prefer spaces around the `-` operator: `this.arr[this.arr.length - 1]`